docs/src/landing.md

```diff
@@ -0,0 +1,59 @@
+# Welcome to the `ethstaker-deposit-cli`!
+
+The [`ethstaker-deposit-cli`](https://github.com/eth-educators/ethstaker-deposit-cli) is a command-line tool forked from Ethereum's [`staking-deposit-cli`](https://github.com/ethereum/staking-deposit-cli) with more functionality and improved performance.
+
+## Important Concerns
+
+This tool generates and/or utilizes mnemonics and validator keystores. Both are highly sensitive materials and their exposure could lead to the loss or theft of funds. Users must take appropriate measures to secure this information.
+
+Here are some core recommendations:
+
+- **Use this tool offline**: We highly recommend using this tool in an offline environment, preferably air-gapped, using a live USB such as [Tails](https://tails.net/install/download/index.en.html).
+
+- **Avoid using passwords or mnemonics as command line arguments**: When executing any command, you will be prompted for the necessary arguments. This prevents the arguments from being stored in your command history which could create an attack vector.
+
+- **Create backups of your mnemonic**: The mnemonic generated by this tool is crucial to your operation and success as a validator. Losing this mnemonic can put you at significant risk of losing your funds. Follow the instructions carefully when creating a mnemonic, ensure you make a backup copy, and store it in a safe location.
```

Comment:
We could expand on securing backups. Multiple copies in multiple physical locations away from fire, water and theft risks should be the norm or the recommended practice.

Reply:
We could even mention storing the seed in steel with the various steel plate products.

Reply:
Added wording with options for long-term protection